migration_toolkit_for_containers/mtc-direct-migration-requirements.adoc

```diff
@@ -48,6 +48,24 @@ include::modules/configuring-retries-for-rsync.adoc[leveloffset=+3]
 
 include::modules/ocp-running-rsync-root-or-non-root.adoc[leveloffset=+3]
 
+include::modules/security-context-constraints-psa-about.adoc[leveloffset=+4]
+
+.Example privileged namespace configuration
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    openshift.io/cluster-monitoring: "true"
+    pod-security.kubernetes.io/enforce: privileged
+    pod-security.kubernetes.io/audit: privileged
+    pod-security.kubernetes.io/warn: privileged
+  name: "<mig_namespace>"
+# ...
+----
+
 To learn more about Pod Security Admission and setting values for labels, see xref:../authentication/understanding-and-managing-pod-security-admission.adoc#security-context-constraints-psa-opting_understanding-and-managing-pod-security-admission[Controlling pod security admission synchronization].

```

Comment:
Nit/not a merge blocker:
From a mod docs perspective, I think it would be best to include the example configuration in the previous module. If that module is reused, this important bit of clarifying information will be missing. 

Also, a simple additional resources section as a trailing include is sufficient here. The introductory clause isn't adding much value and can be considered fluff. 

All that said, these aren't merge blockers, just considerations for future improvements.